Trellix/trellix-network-security/_meta/smart-descriptions.json

```
@@ -0,0 +1,19 @@
+[
+  {
+    "value": "{event.action} with severity {event.severity} was called from {source.ip} to {destination.ip}",
+    "conditions": [
+      {
+        "field": "event.action"
+      },
+      {
+        "field": "event.severity"
+      },
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "destination.ip"
+      }
+    ]
```

I suggest adding a relationship for this smart-description
```suggestion
    ],
    "relationships": [{
         "source": "source.ip",
         "target": "destination.ip",
         "type": "was connected to"
    }]
```